features/features.go

```diff
@@ -694,4 +694,12 @@ var (
 					enhancementPR("https://github.com/openshift/enhancements/pull/1711").
 					enableIn(configv1.DevPreviewNoUpgrade, configv1.TechPreviewNoUpgrade).
 					mustRegister()
+
+	FeatureGateHighlyAvailableArbiter = newFeatureGate("HighlyAvailableArbiter").
+						reportProblemsToJiraComponent("TwoNode / Arbiter").
```

Comment:
This already exists in OCPBUGS?

Reply:
Yup! We added that component in Jira a few weeks back